packages/fast-usdc-deploy/test/fast-usdc.test.ts

```diff
@@ -442,6 +543,15 @@ test.serial('deploy HEAD; upgrade to support EVM destinations', async t => {
   await documentStorageSchema(t, t.context.storage, doc);
 });
 
+test.serial('unmatched batch transactions are disbursed', async t => {
+  for (const txHash of [
+    '0xRC2TestTxHash1000000000000000000000000000000000000000000000000',
+    '0xRC2TestTxHash2000000000000000000000000000000000000000000000000',
+  ]) {
+    t.like(t.context.readTxnRecord({ txHash }).at(-1), { status: 'DISBURSED' });
+  }
```

Comment:
Nice. I believe `publishTxnRecord` is the last step of `settler.self.disburse`, so we should have confidence the pool is actually repaid.

Nonetheless, worth checking `poolMetrics.encumberedBalance` decreases by `evidence1Amount` + `evidence2Amount`?